learn/core_concepts/primary_key.md

```diff
@@ -141,40 +141,40 @@ The code below updates the primary key to `title`:
 
 ### Meilisearch guesses your primary key
 
-Suppose you add documents to an index without previously setting its primary key. In this case, Meilisearch will automatically look for an attribute containing the string `id` in a case-insensitive manner (for example, `uid`, `BookId`, `ID`, `123id123`) in your first document and set it as the index's primary key.
+Suppose you add documents to an index without previously setting its primary key. In this case, Meilisearch will automatically look for an attribute ending with the string `id` in a case-insensitive manner (for example, `uid`, `BookId`, `ID`) in your first document and set it as the index's primary key.
 
-If Meilisearch cannot find a suitable attribute, the document addition process will be interrupted and no documents will be added to your index.
+If Meilisearch finds multiple attributes ending with `id`, it will throw the [`index_primary_key_multiple_candidates_found`](#index-primary-key-multiple-candidates-found) error and ask you to manually set the primary key. If it cannot find a suitable attribute, it will throw the [`index_primary_key_no_candidate_found`](#index-primary-key-no-candidate-found) error. In both cases, the document addition process will be interrupted and no documents will be added to your index.
```

Comment:
I felt that the error code names are unnecessary here, since they make the text more congested and are also mentioned further down on the page. Please push back if you disagree.

```suggestion
If Meilisearch finds [multiple attributes ending with `id`](#index-primary-key-multiple-candidates-found) or [cannot find a suitable attribute](#index-primary-key-no-candidate-found), it will throw an error. In both cases, the document addition process will be interrupted and no documents will be added to your index.
```